cpp/modmesh/math/Complex.hpp

```diff
@@ -35,52 +35,57 @@ namespace modmesh
 
 namespace detail
 {
-template <typename T, typename = std::enable_if_t<std::is_arithmetic<T>::value>>
+
```

Comment:
Add a blank line.  It makes the sectioning looks better (in my personal opinion).  I did it in other places in the code base.